### ISSUE TITLE
If check - False positive - VIP.ValidatedSanitizedInput

### ISSUE THREAD

**Issue by grappler** [WordPress-Coding-Standards/WordPress-Coding-Standards#650]
When running the sniff there is an error for `if ( 'page' === $_POST['post_type'] ) {`

I don't think `$_POST['post_type']` needs to be escaped as if the data was insecure it would fail the check.

**Comment by westonruter** [comment]
The issue the sniff is raising is you need to do an `isset()` check to avoid a possible PHP notice. This should fix the problem:

```php
if ( isset( $_POST['post_type'] ) && 'page' === $_POST['post_type'] ) {
```

**Comment by grappler** [comment]
Ok, after re-looking at it, it makes sense. I think we will need to have explanations and documentation on the error messages for the theme review theme check.
